aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -150,6 +187,167 @@ module aptos_framework::coin {
     /// Capability required to burn coins.
     struct BurnCapability<phantom CoinType> has copy, store {}
 
+    /// The mapping between coin and fungible asset.
+    struct CoinConversionMap has key {
+        coin_to_fungible_asset_map: Table<TypeInfo, address>,
```

Comment:
Can you add some comments here as to why Table over SmartTable? I assume it's because of incremental gas cost affecting others when more coins are added to the same bucket